app/views/reports/regions/_index_tree.html.erb

```diff
@@ -0,0 +1,75 @@
+<div id="facility-access" data-page="show">
+
+  <% regions.each do |org, regions_by_district| %>
+
+  <div class="card access-tree show">
+    <div class="access-tree__section-header">
+      <%= org.name %>
+    </div>
+
+    <ul>
+      <% regions_by_district.each do |district, regions_by_block| %>
+      <li>
+        <div class="access-item">
+
+          <div class="access-item__dropdown">
+            <i class="fas fa-caret-down light"></i>
+          </div>
+
+          <div class="form-check show">
+            <%= link_to_if(accessible_district?(district), district.name, reports_region_path(district, report_scope: district.region_type)) %>
```

Comment:
`link_to_if` to use the permissions cache to see if we should like to the region or just show it as a navigation guide.